atomsci/ddm/pipeline/model_pipeline.py

```diff
@@ -301,7 +301,7 @@ def load_featurize_data(self, params=None):
         # is fitted to the training data only. The transformers are then applied to the training,
         # validation and test sets separately.
         if not params.split_only:
-            self.model_wrapper.create_transformers(self.data)
+            self.model_wrapper.create_transformers(trans.get_all_training_datasets(self.data))
```

Comment:
this `get_all_training_datasets` seems like a function that belongs in `model_datasets`?

Reply:
I disagree. The structure created by get_all_training_datasets is very specific to how transformers are implemented and I would like all that code in one place. Eventually I think there should be a TransformerManager object that can build all needed transformers given a dataset.